### PR TITLE
Follow unfollow

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added follow and unfollow functionality
- Added Filter post functionality

### CODE REVIEW

File: src/Features/User/Components/Profile.js [hunk 1]
```diff
@@ -9,13 +9,17 @@ import ProfileEditModal from "./ProfileEditModal";
...
+  const { username, email, following, followers } = useSelector(
+    (store) => store.auth
+  );
   const { posts, status } = useSelector((store) => store.posts);
 
   useEffect(() => {
     dispatch(getUsersPosts({ username }));
   }, [dispatch, username]);
 
+  console.log(followers);
```
Comment (msdeshmukh009) [inline src/Features/User/Components/Profile.js]: You can remove this console

**Review by msdeshmukh009 — APPROVED** [review]
Follow/unfollow is working fine. Not sure about the filter as I was not able to see the liked post at the top even though sorting using trending.
Suggestion: Restrict all the routes if the user is unauthenticated.

File: src/Features/User/Components/Profile.js (hunk 1 above)
Comment (vijaytembugade) [reply src/Features/User/Components/Profile.js]: You can check , by liking one post and keep another as it it is. Its working.

**Review by preethasri — APPROVED** [review]
good to merge

**Pull request merged**